macro/src/lib.rs

```diff
@@ -83,25 +85,32 @@ pub fn derive_mmio(input: proc_macro::TokenStream) -> proc_macro::TokenStream {
         None
     } else {
         Some(quote! {
-            #[doc = "Create a new handle to this peripheral given an address."]
-            #[doc = ""]
-            #[doc = "# Safety"]
-            #[doc = ""]
-            #[doc = "See the safety notes for [`new_mmio`]."]
+            /// Create a new handle to this peripheral given an address.
```

Comment:
I swear I tried this syntax and quote! didn't like it. But the docs look OK, so. This is easier to read.